packages/api-clients/open-api/eserviceTemplateApi.yml

```diff
@@ -118,6 +118,9 @@ paths:
       responses:
         "200":
           description: EService template created
+          headers:
+            X-Metadata-Version:
+              $ref: "#/components/headers/MetadataVersionHeader"
           content:
```

Comment:
As done for the descriptor here we should return `CreatedEServiceTemplateVersion` to be able to know which one was the new version created on the M2M side

See https://github.com/pagopa/interop-be-monorepo/blob/release/2.7.0/packages/api-clients/open-api/catalogApi.yml#L580